### ISSUE TITLE
GUI

### ISSUE THREAD

**Issue by etoumey** [jtoumey/powerBasedCC#1]
We need to pick a module for the GUI. I know this is ahead of our current development, but I thought I'd open some issues while I wait for ANT+ stick.

Here are some of the modules I've dug up. I looked mainly for RPi suggested modules.
* PyQt
* Tkinter
* PyGUI

**Comment by jtoumey** [comment]
Sweet. I have seen some references to `Tkinter` while trying to figure out what type of approach to take. 
* Check out this [pyQT vs tkinter](http://stackoverflow.com/questions/1094155/developing-guis-in-python-tkinter-vs-pyqt) stackoverflow discussion.

I cannot find any comparisons of the first two with `PyGUI`.

**Comment by jtoumey** [comment]
Sounds like `pyQT` is the way to go. 

As a test, we could structure a GUI that just has several display fields (un-editable). Then we feed these display fields random numbers to test the display. 

There is not really a whole lot of variation in the way different models display output....
* **PC8**
![](http://i130.photobucket.com/albums/p250/cecilhayduke/20150826_084434.jpg)
* **Elemnt**
![](http://www.wahoofitness.com/media/catalog/product/cache/1/image/9df78eab33525d08d6e5fb8d27136e95/g/p/gps_bike_computer_elemnt_front_v2.jpg.pagespeed.ce.CFlhdSxdRC.jpg)

* **Padrone**
![](http://ep.yimg.com/ay/trisports/cateye-padrone-smart-cycling-computer-3.jpg)

**Comment by etoumey** [comment]
I agree. It would be good to have a little more creative display for W' but we should start with basics.
